components/feedstock-card.js

```diff
@@ -77,38 +75,43 @@ export const FeedstockCard = ({ feedstock }) => {
           position: 'relative',
         }}
       >
-        <Box sx={{ height: '60px', mb: '20px' }}>
-          {tags?.length > 0 && <Tags tags={tags} demo={demo} />}
-        </Box>
         <Box sx={{ flex: 1, position: 'relative' }}>
           <Thumbnail
             url={thumbnail ?? fallbackThumbnail}
             demo={demo}
             color={color}
+            tags={tags}
           />
         </Box>
 
         <Box>
-          <Box mt={3}>
-            <Text sx={{ fontSize: 3, fontWeight: 400 }}>{title}</Text>
+          <Box
+            sx={{
+              fontSize: [3, 3, 4, 4],
```

Comment:
Everything is looking rather large on laptop screens. If you notice in `components`, we often special-case the largest breakpoint, and then use the same font-sizes for everything below that (up to and including laptop screens).

When I recommended adjusting the font-size at the largest breakpoint, I meant doing something like the following:
```suggestion
              fontSize: [3, 3, 3, 4],
```

What do you think about consistently updating everywhere we have `styleName: [x, x, y, y,]` to `styleName: [x, x, x, y]`?

Reply:
great tip. i hadn't noticed the pattern. thank you